Cineaste/Custom UI/SearchController.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  SearchController.swift
+//  Cineaste
+//
+//  Created by Felizia Bernutz on 14.07.18.
+//  Copyright © 2018 notimeforthat.org. All rights reserved.
+//
+
+import UIKit
+
+public class SearchController: UISearchController {
+    public override init(searchResultsController: UIViewController?) {
+        super.init(searchResultsController: searchResultsController)
+
+        setup()
+    }
+
+    required public init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
```

Comment:
Can be removed if not needed

Reply:
But we should always call the setup function regardless of the init, shouldn't we?

Reply:
This is a required initializer of `UISearchController`, so it can't be removed. 

We should only call the setup for the styling one time and the initializer is a perfect place for this as it is only called once for the instance. We don't need the setup styling function in `init(coder:)` because we definitely initialize the `SearchController` with its initializer programmatically.